### PR TITLE
Dummy webpage

### PR DESCRIPTION
**Pull request by archa-agrawal**
created basic html and css file

### CODE REVIEW

File: todo-list-fe.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+div {
```
Comment (vnikhra) [inline todo-list-fe.css]: always use classes

File: todo-list-fe.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+.textBox {
+    box-sizing: content-box;
+    width: 350px;
+    height: 60px;
+    border: 2px solid black;
+    margin-left: 500px; 
+    
+}
+button {
```
Comment (vnikhra) [inline todo-list-fe.css]: same as above

File: todo-list-fe.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <title>List of Todos</title>
+    <link href="./todo-list-fe.css" type="text/CSS" rel="stylesheet">
+</head>
+<body>
+    <div class="mainlist">
+        <h2>List of Todos</h2>
+        
+    </div>
+    <section>
+        <p class="textBox"></p>
```
Comment (vnikhra) [inline todo-list-fe.html]: use <input> for all 3

**Review by vnikhra — CHANGES_REQUESTED** [review]

**Comment by archa-agrawal** [comment]
Screen Shot 2022-05-31 at 3 24 36 PM

**Review by vnikhra — APPROVED** [review]

**Pull request merged**